src/ensembl/src/content/app/entity-viewer/gene-view/_constants.scss

```diff
@@ -1,4 +1,5 @@
-$left_column: 178px;
```

Comment:
Maybe create a `styles` folder at this level to put the constants file in?

<img src="https://user-images.githubusercontent.com/6834224/92143694-874a4b00-ee0d-11ea-8982-cbbc6c2605f3.png" width="300" />

Reply:
However, why did you decide to move these constants to the top level of the gene view? Aren't they tightly related specifically to the grid of the transcript list?

Reply:
This is used in Geneview overview image grid too.